Lib/test/test_urllib2_localnet.py

```diff
@@ -660,6 +663,38 @@ def test_line_iteration(self):
                              (index, len(lines[index]), len(line)))
         self.assertEqual(index + 1, len(lines))
 
+    def test_download_to_stdout(self):
+        content = b"My hovercraft is full of eels."
+        handler = self.start_server([(200, [], content)])
+        proc = subprocess.run(
+            [sys.executable, "-m", "urllib.request",
+            "http://localhost:%s" % handler.port],
```

Comment:
Can you switch this to an f-string? I don't want to add %-formatting going forward.